lambda/getBinInterval.ts

```diff
@@ -3,6 +3,7 @@ import {
 	type TimestreamQueryClient,
 } from '@aws-sdk/client-timestream-query'
 import { parseResult } from '@bifravst/timestream-helpers'
+import { getQueryString } from './getQueryString.js'
 
 export const getBinInterval =
```

Comment:
**problem:** The function name is misleading. It is not return a bin interval, but a list of records for a given interval.